components/director/internal/domain/apptemplate/resolver.go

```diff
@@ -617,29 +605,13 @@ func (r *Resolver) RegisterApplicationFromTemplate(ctx context.Context, in graph
 		return nil, err
 	}
 
-	region, subaccountID, systemFieldDiscovery, err := r.areSystemFieldDiscoveryPrerequisitesAvailable(ctx, appTemplate.ID)
-	if err != nil {
-		return nil, err
-	}
-
-	systemFieldDiscoveryLabelIsTrue := false
-	if systemFieldDiscovery {
-		appCreateInputModel.Webhooks, systemFieldDiscoveryLabelIsTrue = r.systemFieldDiscoveryEngine.EnrichApplicationWebhookIfNeeded(ctx, appCreateInputModel, systemFieldDiscovery, region, subaccountID, appTemplate.Name, applicationName)
-	}
-
 	log.C(ctx).Infof("Creating an Application with name %s from Application Template with name %s", applicationName, in.TemplateName)
-	id, err := r.appSvc.CreateFromTemplate(ctx, appCreateInputModel, &appTemplate.ID, systemFieldDiscoveryLabelIsTrue)
+	id, err := r.appSvc.CreateFromTemplate(ctx, appCreateInputModel, &appTemplate.ID, false)
```

Comment:
rename the name of the variable in the `ApplicationService interface` from `systemFieldDiscoveryLabelIsTrue` to `systemFieldDiscoveryValue`